PENDING.md

```diff
@@ -85,6 +84,7 @@ IMPROVEMENTS
     * [cli] \#1632 Add integration tests to ensure `basecoind init && basecoind` start sequences run successfully for both `democoin` and `basecoin` examples.
     * [store] Speedup IAVL iteration, and consequently everything that requires IAVL iteration. [#2143](https://github.com/cosmos/cosmos-sdk/issues/2143)
     * [simulation] Make timestamps randomized [#2153](https://github.com/cosmos/cosmos-sdk/pull/2153)
+    * [x/stake] \#2139 Add stake queriers for increased performance of Gaia-lite endpoints
```

Comment:
We should add a more descriptive PENDING description here - for instance we could list in sub-bullets the new types of queries which are available

Reply:
There are no new types of queriers available. But I can update the description